### ISSUE TITLE
Wiki pages are not getting merged

### ISSUE THREAD

**Issue by anieminen** [ndunand/moodle-tool_mergeusers#151]
Hi,

I have had a couple of cases where user's pages in Wiki activity were not merged. 
Here is a part of a merging log related to wiki database tables. 5952 is the new user's id (user kept). 
```
UPDATE mdl_wiki_pages SET userid = '5952' WHERE id IN (40)
DELETE FROM mdl_wiki_subwikis WHERE userid = '5952'
UPDATE mdl_wiki_versions SET userid = '5952' WHERE id IN (206, 207, 246)
```

It seems that wiki pages were transferred to the new user id, but I can still find subwikis belonging to the removed user id in the database. 

According to course logs both users had content in the wiki activity. How is the tool supposed to work in this situation?

**Comment by jpahullo** [comment]
Hi!

Please, could you specify de Moodle version and plugin version you are
using please?

The logs seems a bit weird, without more context.

The behavior should be as always: kept user has all activity of both users.

Thanks!

----------
Jordi

**Comment by anieminen** [comment]
Hi,

Moodle version is 3.5.5 and plugin version is the latest, `1.18 for Moodle 3.4-3.5 (Build: 2018050401)`

**Comment by jpahullo** [comment]
the problem is about compound indexes on wiki tables. working on it.

**Comment by jpahullo** [comment]
this is addressed by PR #158
